content/pull-requests/committing-changes-to-your-project/viewing-and-comparing-commits/comparing-commits.md

```diff
@@ -56,7 +56,7 @@ To learn more about other comparison options, see "[AUTOTITLE](/pull-requests/co
 
 You can compare your base repository and any forked repository. This is the view that's presented when a user performs a Pull Request to a project.
 
-To compare branches on different repositories, preface the branch names with user names. For example, by specifying `octocat:main` for `base` and `octo-org:main` for `compare`, you can compare the `main` branch of the repositories respectively owned by `octocat` and `octo-org`.
+To compare branches on different repositories, preface the branch names with user names. For example, by specifying `octocat:main` for `base` and `octo-org:main` for `compare`, you can compare the `main` branch of the repositories respectively owned by `octocat` and `octo-org`. You can also preface the branch name with a user name and a repository name. For example, specifying `octocat:awesome-app:main` would use the `main` branch in the `octocat/awesome-app` repository. This can be useful in large organizations, where you might have an upstream repository and a fork both owned by the organization. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)."
```

Comment:
The paragraph looks a bit long so maybe the newly added sentences can start its own paragraph?

Reply:
@muzimuzhi Agreed! Feel free to update and I'll get this merged 💛